Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,260 @@
+        {
+            var methods = type.GetMethods(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance | BindingFlags.DeclaredOnly)
+                .Where(m => !m.IsGenericMethod && m.GetMethodBody() != null && !m.MethodImplementationFlags.HasFlag(MethodImplAttributes.InternalCall))
+                .OrderBy(m => m.Name)
+                .ToList();
+
+            return methods.Select(m => new TreeViewItem(GetMethodSignature(m), m)).ToList();
+        }
+        private string GetMethodSignature(MethodInfo method)
+        {
+            var sb = new StringBuilder();
+            sb.AppendTypeName(method.ReturnType);
+            sb.Append(" ");
+            sb.AppendColored(method.Name, "#dcdcaa");
+            sb.AppendColored("(", "#efb839");
+            var parameters = method.GetParameters();
+            for (int i = 0; i < parameters.Length; i++)
+            {
+                if (i > 0) sb.Append(", ");
+                sb.AppendTypeName(parameters[i].ParameterType);
+                sb.Append(" ");
+                sb.Append(parameters[i].Name);
+            }
+            sb.AppendColored(")", "#efb839");
+            return sb.ToString();
+        }
+        private static unsafe string GetDisassembly(MethodBase method)
+        {
+            if (!JitInspectorHelpers.TryGetJitCode(method, out var code, out var size))
+                return string.Empty;
+
+            return GetDisassembly(method, (byte*)code, size, new NasmFormatter
+            {
+                Options =
+                {
+                    FirstOperandCharIndex = 10
+                }
+            });
+        }
+        private static unsafe string GetDisassembly(MethodBase method, byte* code, int size, Formatter formatter)
+        {
+            using var text = new StringWriter();
+            JitInspectorHelpers.WriteDisassembly(method, code, size, formatter, text);
+            return text.ToString();
+        }
+
+    }
+    public static class StringBuilderExtensions
+    {
+        public static readonly IReadOnlyDictionary<Type, string> SpecialTypeNames = new Dictionary<Type, string>()
+        {
+            { typeof(void)    , "void"   },
+            { typeof(Single)  , "float"  },
+            { typeof(Double)  , "double" },
+            { typeof(Int16)   , "short"  },
+            { typeof(Int32)   , "int"    },
+            { typeof(Int64)   , "long"   },
+            { typeof(UInt16)  , "ushort" },
+            { typeof(UInt32)  , "uint"   },
+            { typeof(UInt64)  , "ulong"  },
+            { typeof(Boolean) , "bool"   },
+            { typeof(String)  , "string" },
+        };
+
+        public static StringBuilder AppendColored(this StringBuilder builder, string text, string color)
+        {
+            return builder.Append("<color=").Append(color).Append(">").Append(text).Append("</color>");
+        }
+        public static string HighlightTypeName(Type type) =>
```

Comment:
I think I'd prefer if this was in `JitInspectorHelpers` and was instead `GetHighlightColor(Type)`. You would then use `AppendColored` to actually write the name. In places where all you need is a `string`, then maybe define a `string GetColoredString(string text, string color)` helper (or just pay for the `StringBuilder` allocation).

Reply:
I believe the changes made meet these requirements

Reply:
Yep, LGTM